src/main/java/com/conveyal/r5/analyst/WebMercatorGridPointSet.java

```diff
@@ -181,6 +200,16 @@ public String getPointId (int i) {
         return x + "," + y;
     }
 
+    public int getPointIndexContaining (Coordinate latLon) {
+        return  getPointIndexContaining(latLon.x, latLon.y);
+    }
+
+    public int getPointIndexContaining (double lon, double lat) {
+        int x = lonToPixel(lon) - west;
+        int y = latToPixel(lat) - north;
+        return y * width + x;
+    }
```

Comment:
Oh right they're nearly identical. It's not clear why `indexFromWgsCoordinates` needs a zoom parameter since it's an instance method on an object that has one specific zoom.